explainers/navigation-to-bundles.md

```diff
@@ -0,0 +1,83 @@
+
+## Fetch with Bundles
+
+If the browsingContext that has a non-null **bundledResources** issues a fetch request that meets the following conditions:
+
+* The request’s method is GET,
+* The request’s URL matches with one of the URLs in the request map in the **bundledResources**, and the [Variants algorithm](https://httpwg.org/http-extensions/draft-ietf-httpbis-variants.html#cache) matches the request headers with the Variants header in the bundle,
+* Either of the following is true:
+  * The request is for same-origin and the requesting URL’s path is restricted by the path of the Bundle URL,
+  * The corresponding response that is in the **bundledResources** has a valid (non-expired) signature, or
+  * The initiator origin is not created as a [Secure Context](https://w3c.github.io/webappsec-secure-contexts/). In this last case the resource is loaded as coming from a unique origin.
+
+Then the UA retrieves the corresponding response from the Bundle and attach it to the request as a [stashed exchange](https://wicg.github.io/webpackage/loading.html#request-stashed-exchange) (Note: we might want to stash the whole bundledResources, but we’re not crystal clear on that idea yet).  Note that the exchange is already a **parsedExchange** here, and only be used in [HTTP-network-or-cache](https://wicg.github.io/webpackage/loading.html#mp-http-network-or-cache-fetch) fetch step (but not in [HTTP fetch](https://wicg.github.io/webpackage/loading.html#mp-http-fetch)).
+
+## Navigation with Inherited Bundles
+If a request is a navigation request or a worker request and if the response was from the BundledResources, the initiator’s bundledResources are copied to the target browsingContext.
+
+## Unique-origin Web Bundle Loading
+When a Bundle without a valid signature is loaded from a unique origin (e.g. a local file), following URL conversion must be performed:
+
+* Set the document’s URL to the unique URL of the Bundle package's one. 
+   * See [@jyasskin](https://github.com/jyasskin)'s [Origins for Resources in Unsigned Packages](https://docs.google.com/document/d/1BYQEi8xkXDAg9lxm3PaoMzEutuQAZi1r8Y0pLaFJQoo/edit#heading=h.1fej4450b9k9) to see what's being proposed.
+   * Chrome Canary 80 has an [experimental implementation of this feature](https://chromium.googlesource.com/chromium/src/+/refs/heads/master/content/browser/web_package/using_web_bundles.md), and it creates a URL for a local Bundle by concatenation of the location (URL) of the Bundle, `?` and the primaryUrl of the Bundle (that must be properly URL-encoded). (e.g. if the primaryUrl is `https://bar.com/article.html`, the document’s URL would be shown as `file:///foo/bar.wbn?https://bar.com/article.html`.)
+   * TODO: also look at what TAG did in the original packaging spec: https://www.w3.org/TR/2015/WD-web-packaging-20150115/#fragment-identifiers
```

Comment:
This is incorporated into https://docs.google.com/document/d/1BYQEi8xkXDAg9lxm3PaoMzEutuQAZi1r8Y0pLaFJQoo/edit, so I think we don't need it here. 

```suggestion
```